### PR TITLE
Fix #3077 remove jqueryui

### LINKED ISSUE #3077: Remove jquery-ui from External Import gui
### Describe the problem

jquery-ui is being removed from codebase

### Describe the solution you'd like

The External Import gui at Administration->Coding->'External Data Loads' has a gui based on jquery-ui. Need to remove the reliance of jquery-ui from this gui. Script is at interface/code_systems/dataloads_ajax.php

### PR DESCRIPTION
**Pull request by stu01509**
Fixes #3077 

#### Short description of what this resolves:
Remove jquery-UI.
Add Bootstrap tooltip, collapse, modal, button style.

### CODE REVIEW

File: interface/code_systems/cqm_valueset_howto.php [hunk 1]
```diff
@@ -23,19 +23,23 @@
 
 
 require_once("../../interface/globals.php");
+use OpenEMR\Core\Header;
```
Comment (bradymiller) [inline interface/code_systems/cqm_valueset_howto.php]: Just for clarity in coding, generally have a blank line between the require_once section and the use section.

File: interface/code_systems/dataloads_ajax.php [hunk 2]
```diff
@@ -36,323 +36,253 @@
...
+
+            $.ajax({
+                url: 'list_staged.php',
+                data: parm,
+                cache: false,
+                success: function(data) {
+                    $(stg_load_id).hide();
+                    $(stg_dets_id).html(data);
+                    $(`#${dbName}_instrmsg`).hover( function() {
+                        dlgopen(`${dbName}_howto.php`, '', 800, 250, false, `<div class='text-warning'>${dbName} Installation Details</div>`, {
```
Comment (bradymiller) [inline interface/code_systems/dataloads_ajax.php]: need to translate:
```
dlgopen(`${dbName}_howto.php`, '', 800, 250, false, `<div class='text-warning'>${dbName} <?php echo xla("Installation Details"); ?></div>`, {
```

File: interface/code_systems/dataloads_ajax.php [hunk 3]
```diff
@@ -36,323 +36,253 @@
...
+
+                        $.ajax({
+                            url: 'standard_tables_manage.php',
+                            data: parm,
+                            cache: false,
+                            success: function(data) {
+                                const stg_load_id = `#${dbName}_stg_loading`;
+                                $(stg_load_id).hide();
+
+                                dlgopen('', '', 800, 250, '', `<div class='text-success'>Successfully upgraded ${dbName}</div>`, {
```
Comment (bradymiller) [inline interface/code_systems/dataloads_ajax.php]: Need to translate:
```
dlgopen('', '', 800, 250, '', `<div class='text-success'><?php echo xla("Successfully upgraded"); ?> ${dbName}</div>`, {
```

**Comment by bradymiller** [comment]
Noted the following bug when trying to open a tooltip:
image

This is because the scripts are lowercase, but the db labels are in upper case so:
1. Windows will likely work fine since casing is ignored for files
2. Linux will break since casing is not ignored for files

Solution will be to run function to lowercase the db labels for use in the filename

**Comment by bradymiller** [comment]
Review complete. Just noted couple issues. Looking forward to next revision :)

**Comment by stu01509** [comment]
Hi @bradymiller 

I fixed it, Please review again.
Thank you :)

**Comment by bradymiller** [comment]
hi @stu01509 , Nice job! Testing well and bringing into codebase. jquery-ui's is nearing its end in the OpenEMR codebase :) Thanks for the improvement! -brady

**Comment by stu01509** [comment]
Hi @bradymiller 

Thanks for your review.

I can't believe we remove jquery-UI so faster,
@im-Amitto helped a lot of on remove jquery-UI 👍
